### ISSUE TITLE
Docker issue

### ISSUE THREAD

**Issue by stuartmaxwell** [writeas/writefreely#139]
### Describe the bug
Hi there - I followed the Docker instructions properly but after running `docker-compose up -d` I am unable to run the setup script since the web container never launches properly. A look at the docker logs shows the following errors:
```
2019/07/17 08:22:30 Loading config.ini configuration...
ERROR: 2019/07/17 08:22:30 log.go:26: Unable to load configuration: open config.ini: no such file or directory
```
I've checked the docker-compose file to make sure that the `config.ini.example` file is mapped properly: `- "./config.ini.example:/go/src/app/config.ini"`

Not sure what to check next?

Thanks for open-sourcing this - keen to check it out.

### Steps to reproduce (if necessary)
1. Follow the docker instructions
2. Run `docker-compose up -d`
3. Attempt to run the `./docker-setup.sh` command

### Expected behavior
Setup script should run

### Application configuration
 - **Single mode or Multi-user mode?** - haven't got this far yet
 - **Database?** [mysql/sqlite] - The docker-compose file uses mysql by default - the database container starts up fine
 - **Open registration?** [yes/no] - haven't got this far yet
 - **Federation enabled?** [yes/no] - haven't got this far yet

**Version or last commit**:
Checked out the latest GitHub code: `Latest commit 71fb635 9 days ago`

**Comment by thebaer** [comment]
Thanks for the report. It looks like the app isn't finding a `config.ini` file in the directory `writefreely` is running from. But I don't actually have any experience with Docker -- this part of WriteFreely is maintained entirely by the community -- so I'll have to defer this to anyone else with experience there. Any help would be much appreciated!

**Comment by robjloranger** [comment]
I can check this out

**Comment by thebaer** [comment]
Sounds great.

**Comment by robjloranger** [comment]
@stuartmaxwell sorry about this, I tried to build the docker file and run the compose script with not luck either. Both fail to even build.

I will start by writing a working docker file and we can iterate on a production level compose after that.

**Comment by stuartmaxwell** [comment]
Thanks for looking into this, I'm keen to help too but have been short on time recently and have no experience with Go. Let me know if I can help with any testing though.

**Comment by robjloranger** [comment]
@stuartmaxwell if you are keen to try out writefreely before we get these things sorted out on master here. I have some images built from https://github.com/writeas/writefreely/tree/docker on my personal docker hub https://cloud.docker.com/u/robjloranger/repository/docker/robjloranger/writefreely.

There are instructions there and two compose files as well, but they may change.

**Comment by adamelliotfields** [comment]
I believe the instructions are incorrect. The working directory is `/go`, so that means `config.ini` must be mounted at `/go/config.ini`, not `/go/src/app/config.ini`.

The `src/app` path might be used during development.

I'm still working on getting this up and running (I have a different error), but will update this comment with my instructions later (gotta run now).

**Comment by thebaer** [comment]
Sorry this issue went unattended. Would anyone be interested in submitting a fix here, as a pull request?

**Comment by thebaer** [comment]
@shleeable is any of this addressed in #357?
